recommend/recommend.go

```diff
@@ -17,11 +17,12 @@ import (
 
 // Options for karmor recommend
 type Options struct {
-	Images     []string
-	Uselabels  []string
-	Tags       []string
-	Outdir     string
-	Reportfile string
+	Images       []string
+	Uselabels    []string
+	Tags         []string
+	Usenamespace string
+	Outdir       string
+	Reportfile   string
```

Comment:
Naming should be always in UpperCamelCase. `UseLabels`, `UserNamespace`, `OutDir` and `ReportFile`.